core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -56,6 +56,17 @@ oppia.factory('CollectionRightsBackendApiService', [
             _setCollectionStatus(
               collectionId, collectionVersion, true, resolve, reject);
           });
+        },
+
+        /**
+        * Updates a collection's rights to be have private learner access, given
```

Comment:
Please ensure the asterisks are aligned for both of these lines of the documentation comment.